src/console/__snapshots__/ConsolePage.test.jsx.snap

```diff
@@ -168,7 +168,6 @@ exports[`ConsolePage renders program banners when they are included 1`] = `
   <div>
     <nav
       aria-label="pagination navigation"
-      className={undefined}
```

Comment:
Why is this being removed here? Are we expecting it to not be in the code anymore?

Reply:
I'm actually not entirely sure why these are now missing from the snapshot. Everything looks fine when I spin it up, but I can look into it more.